handbook/ce/support-values-enablement.md

```diff
@@ -0,0 +1,131 @@
+* Read the introduction in Pamela Fuller’s book *The Leader’s Guide to Unconscious Bias*
+* Now look at what you wrote down for #1 … make note of what aligns with the definitions offered in the book vs what differs.
+* Reflect on what you hope to take away from this training and also reflect on what makes you feel even a small amount of discomfort or uncertainty. 
+
+The purpose of this session is to get the cohort comfortable working together and also for the facilitator to make notes to be supportive of each cohort member.
+
+Agenda for session 1:
+
+* (10 min) Introduction round robin: Name, role at Sourcegraph, something from the last week that brought you joy
+* (10 min) Framing: Facilitator walks through the reason why we do this training and sees if anyone has any questions
+* (15 min) Sharing round robin 1: Did you find that you were defining bias and unconscious bias the same or different from the definitions Fuller offers?
+* (15 min) Sharing round robin 2: What do you hope to take away from this training? (Facilitator shares they will write these down so we can review in our last session)
+* (15 min) Sharing round robin 3: What makes you feel even a small amount of discomfort or uncertainty? 
+* (20 min) Open Q&A -- anything goes; if the facilitator doesn’t know the answer, they own following up after
+* (5 min) For session 2 (this is also posted in the Slack channel and the facilitator posts a reminder a few days before the next session, too)...
+	* Read Part 1 of Pamela Fuller’s book The Leader’s Guide to Unconscious Bias (chapters 1-4) and make note of any questions that come to mind as you read.
+	* Complete the worksheets at the end of each chapter (pages 30-33, 44-46, 57-60, and 69-71) and again make note of any questions that come to mind as you complete the worksheets.
+	* Reflect on what you feel you gained from reading these chapters. Make note of 2-3 things you gained.
+
+### Part 2 - Session 2
+Pre-work to be done **before** session 2:
+
+* Read Part 1 of Pamela Fuller’s book The Leader’s Guide to Unconscious Bias (chapters 1-4) and make note of any questions that come to mind as you read.
+* Complete the worksheets at the end of each chapter (pages 30-33, 44-46, 57-60, and 69-71) and again make note of any questions that come to mind as you complete the worksheets.
+* Reflect on what you feel you gained from reading these chapters. Make note of 2-3 things you gained.
+
+The purpose of this session is to reinforce the reading, allow for group discussion, ensure folks are getting questions answered, to help normalize talking about such things that often feel heavy, and for the facilitator to identify any follow-up conversations needed
+
+Agenda for session 2:	
+
+* (10 minutes) Transition into safe place of cohort round robin: Something from the last week that brought you job
```

Comment:
@virginiaulrich I think this may be a typo for "joy", though "brought you job" does delight me as a phrase. :)

Reply:
(Same on 72, 93, etc.)

Reply:
Thanks!